templates/base.html

```diff
@@ -56,12 +56,10 @@
     </header>
 
     <main role="main" class="flex-shrink-0">
-      <div class="container">
-        <!-- Begin page content -->
-        {% block content %}
-        {% endblock content %}
-        <!-- End page content -->
-      </div>
+      <!-- Begin page content -->
```

Comment:
I think the best way would be to add another `{% block %}` before container so that you don't need to add container to every template that extends the base one. It could be named something like `{% block body-area %}` and then you only need to extend `body-area` and no need to modify all other templates just because of metabase. Or, if the naming is confusing that way, we could name the outer block `content` and the inner block `container-content`, and then all of those other templates would extend `container-content` instead of `content`